setup.cfg

```diff
@@ -1,4 +1,5 @@
 [pycodestyle]
+ignore = E401, W602
```

Comment:
No can do. [pycodestyle doesn't allow ignoring per file](https://github.com/PyCQA/pycodestyle/issues/381). It looks political.